ccd-adapter/build.gradle

```diff
@@ -15,6 +15,7 @@ dependencies {
   compile group: 'org.springframework.boot', name: 'spring-boot-starter', version: versions.springBoot
   annotationProcessor group: 'org.projectlombok', name: 'lombok', version: '1.16.20'
   compileOnly group: 'org.projectlombok', name: 'lombok', version: '1.16.20'
+  compile group: 'com.google.guava', name: 'guava', version: '20.0'
```

Comment:
looks unneeded?

Reply:
ImmutableList is not available in this module